mergify_engine/actions/post_check.py

```diff
@@ -98,6 +99,7 @@ async def _post(
                 str(rmf),
             )
 
+        await hooks.send(ctxt, "action.post_check")
```

Comment:
This tracks failure+success but some hooks track only success.
Do we want to make a difference?

Reply:
I track `the action did something on the pull request`